fhir-persistence-jdbc/src/main/java/com/ibm/fhir/persistence/jdbc/dao/impl/FHIRDbDAOImpl.java

```diff
@@ -43,10 +44,10 @@
  * @author markd
  *
  */
-public class FHIRDbDAOBasicImpl<T> implements FHIRDbDAO {
+public class FHIRDbDAOImpl implements FHIRDbDAO {
```

Comment:
I like the change, it's not always good to use template.

Reply:
yeah, from what i can tell this particular type argument was never even actually used.  it only caused every single reference to it to need a @suppressWarnings annotation :-)